CONTRIBUTING.md

```diff
@@ -0,0 +1,900 @@
+# Bug reports
+
+Use the [issue tracker](https://github.com/lhorie/mithril.js/issues). Do check to make sure your bug hasn't already been filed. Please give the following information, where possible:
+
+1. The version of Mithril you're using, whether it's the dev version or [the version on npm](http://npm.im/mithril.js). The version on npm may not have all the latest bug fixes, so your bug might very well be fixed in the dev version.
+2. The name and version of the browser(s) affected.
+3. A detailed explanation of the bug.
+4. A test case. The simpler, the better.
+
+# Feature requests
+
+Use the [issue tracker](https://github.com/lhorie/mithril.js/issues). Please do the following, where possible:
+
+1. Check to make sure your suggestion hasn't already been filed. There's a nice collection of some of these feature requests [here](https://github.com/lhorie/mithril.js/issues/802).
+2. Clearly denote your issue as a feature request. It helps make the intent clearer. Even better is to denote it in the title.
+3. Describe your idea. This is the most important part.
+4. Submit, and wait for feedback. Don't forget to drop by the [Gitter room](https://gitter.im/lhorie/mithril.js) to get some publicity. It gets traffic daily. :smile:
+
+# Contributing
+
+We welcome any and all contributions. This is a community-driven project. Although we don't have a lot, we do have a few guidelines for contributions.
+
+1. Please adhere to the style guide. It's pretty lengthy, but most of it is checked by ESLint, and if anything fails to check, the build will fail, and you will know about it. Most of it is common practice and common sense. ESLint is also set up to check for other common errors, such as undeclared variables and invalid `typeof` values.
```

Comment:
I'd word this as an FYI, rather than a request to reduce barrier of entry (i.e. "There's a style guide that is checked by ESLint. If you have troubles submitting a pull request, try fixing the ESLint warnings.")